modules/4337/package.json

```diff
@@ -9,29 +9,27 @@
   "files": [
     "contracts",
     "dist",
-    "src",
-    "test",
```

Comment:
Somehow, I ran into an issue where it tried to build the source files inside the 4337 dependency. I don't think we need to publish source and test files to npm, so I removed them.

Reply:
I would argue that including `src` in the NPM package makes sense (it allows you to import TypeScript directly instead of the built JS artifacts).